### DISCUSSION TITLE
片側の基盤のみ、全てのキーソケットが奥まで入らない

### DISCUSSION THREAD

**Discussion by hamusack** [Yowkees/keyball#178] [Q&A] [answered]
お世話になっております。
昨年末にのkeyball61を購入させていただき、ただいま組み立てている最中なのですが
片側基盤のみキーソケットが奥まで入らない状況で組み立ててしまいました。（以下写真）
![IMG_2964](https://user-images.githubusercontent.com/26719451/211125713-ee024212-f5ba-4f16-b35b-abb1398036b1.jpg)

当方、こちらのキーボードが初の自作キーボードとなっており、こういうものなのかと思いながら組み立て続けたところ
もう片側の基盤はスルっとソケットがはまったため、質問させていただいております。

以下、2点ほどご質問にお答えいただけると幸いです。

1. こちらの当方の組み立て方に問題があるのでしょうか…？
また、その場合はどのようにすればソケットがきれいにハマりますでしょうか？
2. こちら、このまま組み立てを進めていった場合、左右のパーツの高さが異なる状況となるのですが
ボトムプレートの取り付けなどで問題となる場合がございますでしょうか？
 
以上、ご回答のほどよろしくお願いいたします。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。
状況了解しました。
この状態ですと、ボトムプレートに干渉してうまく組み立てられないと思いますので一旦組み立てを止めてください。
基板の交換もしくは単体での販売でご相談させていただきたいので、
お手数ですが白銀ラボホームページのお問合せよりメールをお送りいただけますでしょうか。
ご連絡お待ちしております。
